LayoutTests/http/wpt/webauthn/public-key-credential-create-failure.https.html

```diff
@@ -29,69 +29,6 @@
             navigator.credentials.create(options), "Operation timed out.");
     }, "PublicKeyCredential's [[create]] with timeout");
 
-    promise_test(function(t) {
```

Comment:
This is sad but I understand why. I hope we can readd these with some new test development.